app/controllers/accountants_controller.rb

```diff
@@ -1,9 +1,23 @@
 class AccountantsController < ApplicationController
   def index
-    head :ok
   end
 
   def search
-    head :ok
+    @results = Patient.search params[:search]
+
+    @patient = Patient.new
```

Comment:
Yeah I wasn't sure what those all were for. Looked like helpers for if you searched by phone/blah blah. It seems like they should stick around given the mockup?